js/rpc-server/Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG NODE_VERSION
```

Comment:
I think this Docker image should either be based on `runner` or actually `runner`

Reply:
How do we feel about [this](https://github.com/reach-sh/reach-lang/pull/50/commits/abe562ef465a4ed232654d21ebf37f7aefc2379d)?

Reply:
Amen